vite.config.js

```diff
@@ -4,6 +4,7 @@ import path from 'path';
 /** @type {import('vite').UserConfig} */
 const config = {
   plugins: [sveltekit()],
+  optimizeDeps: ['date-fns', 'date-fns-tz'],
```

Comment:
this should be optimizeDeps: {
  include: ['date-fns', 'date-fns-tz']
}

https://vitejs.dev/config/dep-optimization-options.html